realm/realm-jni/build.gradle

```diff
@@ -248,6 +248,67 @@ task deployCore(group: 'build setup', description: 'Deploy the latest version of
     }
 }
 
+task assembleRobolectric(description: "Build the Robolectric shared library for Darwin") {
+    def staticLibPath = "/usr/local/lib/librealm.a"
```

Comment:
Would it be good if it can compile the core from source code if `coreSourcePath` exists?